app/components/StyledCard/index.js

```diff
@@ -158,7 +163,8 @@ function StyledCard(props) {
                 avatar={<CustomerServiceOutlined style={{ fontSize: 20 }} spin={currentSong === index} />}
                 title={data.trackCensoredName}
               />
-              <Meta title={<StyledSmall>{data.artistName}</StyledSmall>} description={showProgressBar(index)} />
+              
+              <Meta title={<Link exact to={`/track/${item.trackId}`}><StyledSmall>{data.artistName}</StyledSmall></Link>} description={showProgressBar(index)} />
```

Comment:
let's get the value of the route from the routeConstants and then replace `:trackId` with the value `item.trackId`